APIs/schemas/output.json

```
@@ -40,7 +41,12 @@
               "type": "string"
             }
           }
-        }
+        },
+        "device_id": {
+          "description": "Device ID which this Output forms part of. This attribute is used to ensure referential integrity by registry implementations.",
```

I don't think this is related to registry implementations, so unless we can think of better explanation, I suggest deleting second sentence.

```suggestion
          "description": "Device ID which this Output forms part of",
```

---

Inputs/Outputs are not stored in the  registry. So I agree to remove this sentence "This attribute is used to ensure referential integrity by registry implementations."

---

Agreed